src/main/java/de/tum/in/www1/artemis/service/TextAssessmentService.java

```diff
@@ -78,6 +78,9 @@ public Result saveAssessment(Long resultId, List<Feedback> textAssessment, TextE
         }
 
         result.setAssessmentType(AssessmentType.MANUAL);
```

Comment:
Now that the assessmentType is not always manual, the doc has become outdated.

`It updates the completion date, sets the assessment type to MANUAL and sets the assessor attribute.`

Reply:
I edited the doc